invenio_vocabularies/cli.py

```diff
@@ -30,7 +31,17 @@ def get_config_for_ds(vocabulary, filepath=None, origin=None):
             with open(filepath) as f:
                 config = yaml.safe_load(f).get(vocabulary)
         if origin:
-            config["reader"]["args"]["origin"] = origin
+            config["readers"][0]["args"]["origin"] = origin
+
+        return config
+
+    if vocabulary == "funders":
```

Comment:
action: create an issue for the creation of a "factory" and remove the above `FIXME`

Reply:
https://github.com/inveniosoftware/invenio-vocabularies/issues/164